# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/glib/hotfix-no-doc.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Hotfix: do not build subdir docs/reference as it errors out and can not be
disabled via meson_options.txt.

  - NoTag <notag@maneto.fr>

--- glib-2.76.2/meson.build.vanilla	2023-05-24 11:34:23.943583213 +0200
+++ glib-2.76.2/meson.build	2023-05-24 11:34:36.192584167 +0200
@@ -2442,7 +2442,6 @@
 endif
 
 gnome = import('gnome')
-subdir('docs/reference')
 
 summary({
   'host cpu' : host_machine.cpu_family(),
